### PR TITLE
Add spect

### PR DESCRIPTION
**Pull request by dy**
4.4kb aspect-oriented dom framework - includes full-featured hyperscript with observables and selector-based aspects.
https://github.com/spectjs/spect

### CODE REVIEW

**Comment by dy** [comment]
Hallou?

**Comment by dy** [comment]
Ping

File: data.js [hunk 1]
```diff
@@ -5922,5 +5922,13 @@ module.exports = [
...
     source: "https://raw.githubusercontent.com/Stuhl/javascript-entity-component-system/master/src/index.js"
-  }
+  },
+  {
+    name: "spect",
+    github: "spectjs/spect",
+    tags: ["aspect", "aop", "frp", "observable", "selector-observer", "hyperscript", "htm"],
+    description: "Aspect-oriented reactive DOM framework",
+    url: "https://github.com/spectjs/spect",
+    source: "https://unpkg.com/browse/spect@22.0.0/dist/spect.min.js"
```
Comment (pineapplemachine) [inline data.js:5932]: This should be a link to raw and unminified source code. Please refer to the repository's readme, which lays out what is expected of submitted packages.

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified

**Review by pineapplemachine — CHANGES_REQUESTED** [review]

File: data.js (hunk 1 above)
Comment (pineapplemachine) [reply data.js:5932]: Sorry for the delay. I've been pretty absorbed with other things. Unfortunately I don't think there's anyone very actively maintaining this repository at the moment.

**Pull request merged**